### ISSUE TITLE
FED_Rockfish_Catch data summarize?

### ISSUE THREAD

**Issue by reikopm** [marinebon/info-mb#2]
The Fisheries Ecology Division (FED) data are not summarized, I'm retrieving the data the ERDDAP. It would be nice to be able to do this before graphing for yearly abundance sums. 

Alternatively one could download the data, sum it and then upload the file to the github. But it would be better to be able to do it on the fly.

![sardine_abundance](https://user-images.githubusercontent.com/1767453/30680309-b24079e0-9e53-11e7-9f61-d1b43a4c64a9.JPG)

**Comment by bbest** [comment]
Yeah, good call. I can make aggregation by year the default in the [`plot_timeseries()`](https://github.com/marinebon/infographiq/blob/master/R/plot_timeseries.R) function.
